src/components/EditCoursePage/index.jsx

```diff
@@ -92,6 +92,7 @@ class EditCoursePage extends React.Component {
         min_effort: courseRun.min_effort ? courseRun.min_effort : null,
         pacing_type: courseRun.pacing_type,
         // Reduce Staff list to just the UUID
+        rerun: courseRun.rerun ? courseRun.rerun : null,
```

Comment:
Move this so the comment still applies to the line directly below it.

Reply:
Good catch